examples/kitchen-sink.yaml

```
@@ -262,6 +262,7 @@ meter_provider:
           included:
             - key1
             - key2
+          # Configure resource attributes to be excluded, in this example attribute service.attr1.
```

```suggestion
          # Configure resource attributes to be excluded, in this example attribute key3.
```
This brings up a good point. It seems like this needs to be _either_ a list to include, OR a list to exclude, but not both?

---

Does it? I would expect this to be similar to how resource attributes can be included or excluded. The example in the kitchen sink shows an include of `service.*` and an exclude of `service.attr1`

---

Not sure how SDKs would handle the presence of both include and exclude lists. I know java (and probably go) can handle.